src/Widgets/calendar/CalendarModel.vala

```diff
@@ -16,6 +16,16 @@
  */
 
 namespace DateTime.Widgets {
+    public enum Weekday {
```

Comment:
Some kind of copy/paste mess up adding this here?

Reply:
It wouldn't compile if this wasn't here.